### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by marcode95**
## Fourth Microverse Project: Apple Homepage Clone

-Added background image properties and gradients
-Added grid and flex to adjust the different elements

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  color: #fff;
+}
+
+header .nav-bar-ul {
+  display: flex;
+  width: auto;
```
Comment (bertil291utn) [inline style.css]: ## Using the Same Rule Repetitively[1]
- [x] Create a class once and the use in each HTML tag where you need it.
- [x] Delete all your `display:flex;` properties in entire CSS file
- [x] Add your HTML tags

**Example:**

**CSS file**

```
.flex{
  display: flex;
}
```
**Html file**

```
<div class="flex">
```
**Note:** Do this change to all properties in this file 

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

File: style.css [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+header .apple-link i {
+  font-size: 18px;
+}
+
+.header-search-form {
+  position: relative;
+  font: 12px/18px "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
+  color: #fff;
+  display: flex;
+  justify-content: center;
```
Comment (bertil291utn) [inline style.css]: ## Using the Same Rule Repetitively[1]
- [x] Create a class once and the use in each HTML tag where you need it.
- [x] Delete all your `color: #fff;` properties in the entire CSS file
- [x] Add your HTML tags

**Example:**

**CSS file**

```
.white-color{
  color: #fff;
}
```
**Html file**

```
<div class="white-color">
```
**Note:** Do this change to all properties in this file  also that you have like `color: white`

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

**Review by bertil291utn — CHANGES_REQUESTED** [review]
**Status:** Changes requested 🟠  :shipit:? Not yet.

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.

### Your work is amazing you master it GitHub Flow you have a professional README file. 

However, in order to accomplish as the actual page, you required to do these changes, listed below:

- [x] Each article should a `<a href="#">` wrapping each one

image


### OPTIONAL

- [x] Encourage the use of the [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) for the dominant content of the `<body>` of a document

### Summary

-  Don't forget to work with inline comments below this comment box.

<hr>

Guys work in the requested changes above and request another review.
Good luck. 

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

File: style.css (hunk 1 above)
Comment (marcode95) [reply style.css]: Deleted all display: flex; properties and summarized them in one class

File: style.css (hunk 2 above)
Comment (marcode95) [reply style.css]: Deleted all color: #fff; properties and summarized them in one class

**Comment by bertil291utn** [comment]
**Status:** Approved ✅

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.



<h2 align="center">Ship it </h2>

<p align="center">image</p>

Please merge to master branch and move on with the next assignment.

You're doing  a great job keep going 💻 🚀 🔥

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

**Pull request merged**